### PR TITLE
[WEB-747]  Stepper component

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
See [WEB-747] for details.

Storybook entry shows intended usage.

[WEB-747]: https://tidepool.atlassian.net/browse/WEB-747

### CODE REVIEW

File: app/components/elements/Stepper.js [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+    steps,
+    themeProps,
+    variant,
+    ...stepperProps
+  } = props;
+
+  let initialActiveStepState = initialActiveStep;
+  let initialActiveSubStepState = initialActiveSubStep;
+
+  const activeStepsHash = location.hash.split('-step-')[1];
```
[the remark this answers is not in the document]
Comment (cbwebdevelopment) [reply app/components/elements/Stepper.js:70]: That's a good point.

[the remark this answers is not in the document]
Comment (cbwebdevelopment) [reply app/components/elements/Stepper.js]: I've changed this to use a unique query param instead of a hash.

**Comment by cbwebdevelopment** [comment]
@krystophv I've made a couple of updates to address the reservations you brought up.  The panel content for a given step is now defined within the step props, and I've updated the history handling to leverage unique query params based off of the supplied `id` instead of the hash.

Can you please have a quick look at the 2 most recent commits (ac89177 and 8d80df2) and let me know if you have any further feedback for them?

**Comment by krystophv** [comment]
@cbwebdevelopment New changes look great to me! Nice work on this.

**Comment by cbwebdevelopment** [comment]
Great.  I'll merge now.  Thanks for the feedback 👍

**Pull request merged**